django_project/users/views.py

```diff
@@ -1,10 +1,27 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from .forms import UserRegisterForm
 
 
 def register(request):
-    return render(request, 'users/register.html', {'title': 'Register'})
+    if request.method == 'POST':
+        form = UserRegisterForm(request.POST)
+        form.fields['username'].label = 'Email'
+        if form.is_valid():
+            form.save()
+            username = form.cleaned_data.get('username')
+            messages.success(request, f'Account Created for {username}! You are now able to log in.')
+            return redirect('login')
+    else:
+        form = UserRegisterForm()
+    return render(request, 'users/register.html', {'form': form})
```

Comment:
Let's remove this code and let Anny handle the registration view. You can just set it back to the `return render(request, 'users/register.html', {'title': 'Register'})` line that was there before.